ServerCore/Pages/Submissions/Index.cshtml.cs

```diff
@@ -55,32 +55,32 @@ public async Task<IActionResult> OnPostAsync(int puzzleId, string submissionText
             SubmissionText = submissionText;
             if (!this.Event.IsAnswerSubmissionActive)
             {
-                return Page();
+                return RedirectToPage();
```

Comment:
This causes really bad perf on this page because reloading has to redo a bunch of queries.

Reply:
Ouch :)